tests/lib/rules/balanced-braces.js

```diff
@@ -0,0 +1,105 @@
+const rule = require('../../../lib/rules/balanced-braces');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const eslintTester = new RuleTester();
+eslintTester.run('balanced-braces', rule, {
+  valid: [
+    '{ test: computed("a", "b", function() {}) }',
+    '{ test: computed(function() {}) }',
+    '{ test: computed("a.test", "b.test", function() {}) }',
+    '{ test: computed("a.{test,test2}", "b", function() {}) }',
+    '{ test: computed("a.{test,test2}", "c", "b", function() {}) }',
+    '{ test: computed("model.a.{test,test2}", "model.b.{test3,test4}", function() {}) }',
+    '{ test: computed("foo.bar.{name,place}", "foo.qux.[]", "foo.baz.{thing,@each.stuff}", function() {}) }',
+    '{ test: computed("foo.bar.{name,place,baz,stuff,test.@each.content}","foo.bar.fields.@each.{name,value}","foo.bar.custom.@each.{question,choice}", function() {})}',
+    '{ test: computed("foo1.bar1.{name1,place1,baz1,stuff1,test1.@each.content1,fields1.@each.{name1,value1},custom1.@each.{question1,choice1}}", function() {})}',
+    '{ test: computed.or("model.{projects,files.@each.{isSaving,test}}", "foo.bar.place") }',
+    '{ test: computed.or("foo.bar.name", "foo.bar.place") }',
+    '{ test: computed.and("foo.bar.name", "foo.bar.place") }',
+    '{ test: Ember.computed.filterBy("a", "b", false) }',
+  ],
```

Comment:
We need some valid examples that allowed unbalanced braces in function calls that are not Ember's `computed` function to make sure those are still allowed.

Reply:
I was not sure about what you meant exactly. I simply added function calls containing String arguments that have unbalanced braces.
Did I understand correctly ?

Reply:
For example, this should be a valid test case, to ensure we avoid false positives:
```
someOtherFunctionCall('a.{{something}')
```

Reply:
Ok great, this is what I did
`    'myFunction("{ key: string }, saving,test}", "b}", false)',
    'myFunction("A string containing curly braces {}}")',`
Should I add your test case or change something in mine? 
Thx :smile_cat:

Reply:
Yours is fine, it looks like it achieves the goal of testing for false positives.

Reply:
:+1:  I resolve this conversation then. I think I applied all the changes you recommended.